notebooks/00_quick_start/wide_deep_model_movielens.ipynb

```diff
@@ -0,0 +1,1388 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<i>Copyright (c) Microsoft Corporation. All rights reserved.</i>\n",
+    "\n",
+    "<i>Licensed under the MIT License.</i>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Wide and Deep Model for Movie Recommendation\n",
+    "<br>\n",
+    "\n",
+    "This notebook shows how to build and test [**wide-and-deep model**](https://arxiv.org/abs/1606.07792)--linear combination of linear and deep NN models--using [TensorFlow high-level Estimator API](https://www.tensorflow.org/api_docs/python/tf/estimator/DNNLinearCombinedRegressor).\n",
+    "\n",
+    "On the [movie recommendation dataset](https://grouplens.org/datasets/movielens/), we quickly demonstrate how to prepare features, build the model, use log-hook to estimate performance while training, export model, and load the saved model.\n",
+    "\n",
+    "We use a preset of hyper-parameters optimized for *MovieLens 1M* dataset we found by utilizing **Azure Machine Learning service**([AzureML or AML](https://azure.microsoft.com/en-us/services/machine-learning-service/)). You can find more details about hyperparameter tuning of wide-and-deep model via AML from [our notebook](../04_model_select_and_optimize/aml_hyperparameter_tuning.ipynb)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Prerequisite\n",
+    "* TensorFlow (version 1.8 or higher) - GPU version is preferable. To easily setup a conda environment with `tensorflow-gpu` package, please follow [SETUP].(https://github.com/Microsoft/Recommenders/blob/master/SETUP.md) "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "sys.path.append(\"../../\")\n",
+    "\n",
+    "import os\n",
+    "import shutil\n",
+    "import itertools\n",
+    "\n",
+    "import tensorflow as tf\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import sklearn.preprocessing\n",
+    "\n",
+    "from reco_utils.common import tf_utils\n",
+    "from reco_utils.dataset import movielens\n",
+    "from reco_utils.dataset.pandas_df_utils import user_item_pairs\n",
+    "from reco_utils.dataset.python_splitters import python_random_split\n",
+    "from reco_utils.evaluation.python_evaluation import (\n",
+    "    rmse, mae, rsquared, exp_var,\n",
+    "    map_at_k, ndcg_at_k, precision_at_k, recall_at_k\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Tensorflow Version: 1.12.0\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
```

Comment:
It's tricky since if someone edit the contents of this notebook in the future and run the cells, the output values will be changed and not aligned w/ the HW description anymore... any suggestions?

Reply:
Good point. Maybe we can add the info at the begging of the notebook about the GPU information.

Reply:
Adding